src/conv/k-g/k_parser.cpp

```diff
@@ -72,12 +79,87 @@ static std::vector<std::string> read_line_node_standard
     const char* line
 ) {
     std::vector<std::string> ret;
+    bool commaSeparated = false;
+
+    if (line != nullptr) {
+	std::string temp;
+
+	for (size_t i = 0; i < strlen(line); ++i) {
+	    if (line[i] == ',' || line[i] == ' ') {

```

Comment:
Spaces can appear in the fixed format data too. Therefore, you cannot use it as a flag for comma-separated(?)

How about using `if (strchr(line, '\') != nullptr)` as criterion?